lib/errorlogger.cpp

```diff
@@ -480,7 +480,11 @@ std::string ErrorMessage::toXML() const
     tinyxml2::XMLPrinter printer(nullptr, false, 2);
     printer.OpenElement("error", false);
     printer.PushAttribute("id", id.c_str());
+    if (!guideline.empty())
```

Comment:
You will need to tweak `ErrorLogger::serialize` and `ErrorLogger::deserialize` also otherwise it will not always work when running multiple threads with `-j2`